x-pack/test/alerting_api_integration/observability/custom_threshold_rule/rate_bytes_fired.ts

```diff
@@ -0,0 +1,268 @@
+          .contain('observability');
+        expect(resp.hits.hits[0]._source).property(
+          'kibana.alert.action_group',
+          'custom_threshold.fired'
+        );
+        expect(resp.hits.hits[0]._source).property('tags').contain('observability');
+        expect(resp.hits.hits[0]._source).property(
+          'kibana.alert.instance.id',
+          'host-0,container-0'
+        );
+        expect(resp.hits.hits[0]._source).property('kibana.alert.workflow_status', 'open');
+        expect(resp.hits.hits[0]._source).property('event.kind', 'signal');
+        expect(resp.hits.hits[0]._source).property('event.action', 'open');
+
+        expect(resp.hits.hits[0]._source).property('host.name', 'host-0');
+        expect(resp.hits.hits[0]._source)
+          .property('host.mac')
+          .eql(['00-00-5E-00-53-23', '00-00-5E-00-53-24']);
+        expect(resp.hits.hits[0]._source).property('container.id', 'container-0');
+        expect(resp.hits.hits[0]._source).property('container.name', 'container-name');
+        expect(resp.hits.hits[0]._source).not.property('container.cpu');
+
+        expect(resp.hits.hits[0]._source)
+          .property('kibana.alert.group')
+          .eql([
+            {
+              field: 'host.name',
+              value: 'host-0',
+            },
+            {
+              field: 'container.id',
+              value: 'container-0',
+            },
+          ]);
+
+        expect(resp.hits.hits[0]._source)
+          .property('kibana.alert.rule.parameters')
+          .eql({
+            criteria: [
+              {
+                comparator: '>=',
+                threshold: [0.2],
+                timeSize: 1,
+                timeUnit: 'm',
+                metrics: [{ name: 'A', field: 'system.network.in.bytes', aggType: 'rate' }],
+              },
+            ],
+            alertOnNoData: true,
+            alertOnGroupDisappear: true,
+            searchConfiguration: { index: 'data-view-id', query: { query: '', language: 'kuery' } },
+            groupBy: ['host.name', 'container.id'],
+          });
+      });
+
+      it('should set correct action variables', async () => {
+        const rangeFrom = moment(startedAt).subtract('5', 'minute').toISOString();
+        const resp = await waitForDocumentInIndex<ActionDocument>({
+          esClient,
+          indexName: ALERT_ACTION_INDEX,
+        });
+
+        expect(resp.hits.hits[0]._source?.ruleType).eql('observability.rules.custom_threshold');
+        expect(resp.hits.hits[0]._source?.alertDetailsUrl).eql(
+          `https://localhost:5601/app/observability/alerts?_a=(kuery:%27kibana.alert.uuid:%20%22${alertId}%22%27%2CrangeFrom:%27${rangeFrom}%27%2CrangeTo:now%2Cstatus:all)`
+        );
+        expect(resp.hits.hits[0]._source?.reason).eql(
+          `Rate of system.network.in.bytes is 0.2 B/s, above the threshold of 0.2 B/s. (duration: 1 min, data view: kbn-data-forge-fake_hosts.fake_hosts-*, group: host-0,container-0)`
```

Comment:
`0.2 B/s, above the threshold of 0.2 B/s` 🤔
We need a ticket to fix `above or equal` reason message.

Reply:
Done